src/serac/infrastructure/accelerator.hpp

```diff
@@ -56,10 +56,12 @@ struct cpu_policy {
 };
 
 /**
- * @brief tag type for signaling that calculations that should be performed on the CPU
+ * @brief tag type for signaling that calculations that should be performed on the GPU
  */
+#if defined(__CUDACC__)
 struct gpu_policy {
 };
+#endif
```

Comment:
I don't believe we want this type to be conditionally defined.

Reply:
I'll add a documentation note that it's just a temporary guard and we can get rid of it after we make the 
evaluation_kernel<policy>